aioshelly/wsrpc.py

```diff
@@ -202,10 +210,16 @@ def connected(self) -> bool:
         """Return if we're currently connected."""
         return self._client is not None and not self._client.closed
 
-    async def call(self, method, params=None, timeout=10):
+    async def call(
+        self, method: str, params: dict[str, Any] | None = None, timeout: int = 10
+    ) -> Any:
         """Websocket RPC call."""
         call = RPCCall(self._next_id, method, params, self._route)
         self._calls[call.call_id] = call
+
+        if not self._client:
+            return
+
```

Comment:
This must happen before the `self._calls` is assigned and added a comment to raise error so it is not needed
```suggestion

```